snewpdag/plugins/EvalMap.py

```diff
@@ -26,14 +25,16 @@ def __init__(self, detector_location, nside, in_field, in_det_field, in_det_list
     self.in_field = in_field
     self.in_det_field = in_det_field
     self.in_det_list_field = in_det_list_field
-    self.cache = {} # { <det> : <TimeSeries or TimeHist> }
+    self.cache = {} # { <det> : <TimeSeries or Hist1D> }
     super().__init__(**kwargs)
 
   def reference_time(self):
     """
+    TODO: fix. No reference time anymore in Hist1D or TimeSeries
+
     Calculate earliest reference time over the detectors in the cache.
     Assume neutrino burst time for each detector is reflected in
-    the TimeHist or TimeSeries reference time.
+    the Hist1D or TimeSeries reference time.
```

Comment:
Hist1D might create confusion with Histogram1D in both plugins and renderers? maybe we should in future not re-use the name at all or use the same for the three?

Reply:
good point. maybe rename some things after this round of changes.